[CI] SearchableSnapshotsBlobStoreCacheIntegTests testBlobStoreCache failing

**Build scan**:

https://gradle-enterprise.elastic.co/s/nyqgoqzxp3xxw

**Repro line**:

`./gradlew ':x-pack:plugin:searchable-snapshots:internalClusterTest' --tests "org.elasticsearch.xpack.searchablesnapshots.cache.blob.SearchableSnapshotsBlobStoreCacheIntegTests.testBlobStoreCache" -Dtests.seed=52F02EBD9CEAF1F4 -Dtests.locale=es-PE -Dtests.timezone=US/East-Indiana -Druntime.java=17`

**Reproduces locally?**:

No

**Applicable branches**:

`master`

**Failure history**:

https://gradle-enterprise.elastic.co/scans/tests?search.relativeStartTime=P7D&search.timeZoneId=Europe/Berlin&tests.container=org.elasticsearch.xpack.searchablesnapshots.cache.blob.SearchableSnapshotsBlobStoreCacheIntegTests&tests.sortField=FAILED&tests.test=testBlobStoreCache&tests.unstableOnly=true

**Failure excerpt**:

```
java.lang.AssertionError: {"file_ext":"cfs","num_files":7,"open_count":12,"close_count":5,"size":{"total_in_bytes":1763710,"min_in_bytes":5327,"max_in_bytes":942865,"average_in_bytes":251959},"contiguous_bytes_read":{"count":162,"sum":78532,"min":13,"max":1024},"non_contiguous_bytes_read":{"count":35,"sum":560,"min":16,"max":16},"cached_bytes_read":{"count":31,"sum":12702,"min":13,"max":1024},"index_cache_bytes_read":{"count":166,"sum":66621,"min":16,"max":1242},"cached_bytes_written":{"count":167,"sum":63732,"min":16,"max":1242,"time_in_nanos":15875426},"direct_bytes_read":{"count":0,"sum":0,"min":0,"max":0,"time_in_nanos":0},"optimized_bytes_read":{"count":0,"sum":0,"min":0,"max":0,"time_in_nanos":0},"forward_seeks":{"small":{"count":47,"sum":3617707,"min":34,"max":941825},"large":{"count":0,"sum":0,"min":0,"max":0}},"backward_seeks":{"small":{"count":17,"sum":7242,"min":17,"max":969},"large":{"count":0,"sum":0,"min":0,"max":0}},"blob_store_bytes_requested":{"count":1,"sum":839,"min":839,"max":839},"lucene_bytes_read":{"count":209,"sum":79284,"min":13,"max":1024},"current_index_cache_fills":0}
Expected: <0L>
but: was <1L>
```

Also in the logs:

```
java.lang.reflect.InaccessibleObjectException: Unable to make field private int java.io.FileDescriptor.fd accessible: module java.base does not "opens java.io" to unnamed module @27ce24aa
	at java.lang.reflect.AccessibleObject.checkCanSetAccessible(AccessibleObject.java:354) ~[?:?]
	at java.lang.reflect.AccessibleObject.checkCanSetAccessible(AccessibleObject.java:297) ~[?:?]
	at java.lang.reflect.Field.checkCanSetAccessible(Field.java:178) ~[?:?]
	at java.lang.reflect.Field.setAccessible(Field.java:172) ~[?:?]
	at org.elasticsearch.xpack.searchablesnapshots.preallocate.Preallocate$FileDescriptorFieldAction.run(Preallocate.java:105) ~[preallocate-8.1.0-SNAPSHOT.jar:8.1.0-SNAPSHOT]
	at org.elasticsearch.xpack.searchablesnapshots.preallocate.Preallocate$FileDescriptorFieldAction.run(Preallocate.java:91) ~[preallocate-8.1.0-SNAPSHOT.jar:8.1.0-SNAPSHOT]
	at java.security.AccessController.doPrivileged(AccessController.java:569) ~[?:?]
	at org.elasticsearch.xpack.searchablesnapshots.preallocate.Preallocate.preallocate(Preallocate.java:49) [preallocate-8.1.0-SNAPSHOT.jar:8.1.0-SNAPSHOT]
	at org.elasticsearch.xpack.searchablesnapshots.preallocate.Preallocate.preallocate(Preallocate.java:32) [preallocate-8.1.0-SNAPSHOT.jar:8.1.0-SNAPSHOT]
	at org.elasticsearch.xpack.searchablesnapshots.cache.shared.SharedBytes.<init>(SharedBytes.java:62) [main/:?]
	at org.elasticsearch.xpack.searchablesnapshots.cache.shared.FrozenCacheService.<init>(FrozenCacheService.java:294) [main/:?]
	at org.elasticsearch.xpack.searchablesnapshots.SearchableSnapshots.createComponents(SearchableSnapshots.java:341) [main/:?]
	at org.elasticsearch.xpack.core.LocalStateCompositeXPackPlugin.lambda$createComponents$0(LocalStateCompositeXPackPlugin.java:218) [x-pack-core-test-artifacts-8.1.0-SNAPSHOT.jar:8.1.0-SNAPSHOT]
```

--- a/x-pack/plugin/searchable-snapshots/src/internalClusterTest/java/org/elasticsearch/xpack/searchablesnapshots/cache/blob/SearchableSnapshotsBlobStoreCacheIntegTests.java
+++ b/x-pack/plugin/searchable-snapshots/src/internalClusterTest/java/org/elasticsearch/xpack/searchablesnapshots/cache/blob/SearchableSnapshotsBlobStoreCacheIntegTests.java
@@ -120,7 +120,6 @@ public class SearchableSnapshotsBlobStoreCacheIntegTests extends BaseFrozenSearc
         return Settings.builder().put(super.nodeSettings(nodeOrdinal, otherSettings)).put(cacheSettings).build();
     }
 
-    @AwaitsFix(bugUrl = "https://github.com/elastic/elasticsearch/issues/82470")
     public void testBlobStoreCache() throws Exception {
         final String indexName = randomAlphaOfLength(10).toLowerCase(Locale.ROOT);
         createIndex(indexName);

